python/tidy/servo_tidy/tidy.py

```diff
@@ -723,13 +724,40 @@ def parse_config(content):
     config['ignore']['directories'] = map(lambda path: os.path.join(*path.split('/')),
                                           config['ignore']['directories'])
 
+    # Add dict of dir, list of expected ext to config
+    dirs_to_check = config_file.get("check_ext", {})
+    # Fix the paths (OS-dependent)
+    for path, exts in dirs_to_check.items():
+        fixed_path = os.path.join(*path.split('/'))
+        config['check_ext'][fixed_path] = exts
+
     # Override default configs
     user_configs = config_file.get("configs", [])
     for pref in user_configs:
         if pref in config:
             config[pref] = user_configs[pref]
 
 
+def check_directory_files(directories, print_text=True):
+    if print_text:
+        print '\rChecking directories for correct file extensions...'
+    for directory, file_extensions in directories.items():
+        files = os.listdir(directory)
+        for filename in files:
+            full_path = os.path.join(directory, filename)
+            if not os.path.isfile(full_path):
+                continue
```

Comment:
Um, shouldn't we be throwing an error here? Or else, there wouldn't be any reason to do this check (since directories won't have the extension suffix anyway)